config/pitest-suppressions/pitest-java-ast-visitor-suppressions.xml

```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why are we forced to kill eight mutations in one PR? It would be better to separate out mutations that are killed by test only to another PR.

For removal of code, we usually demand a single PR per mutation.

Reply:
It was just because they were simple fixes. I had to plan to split out more if I had to do more removals.

Reply:
Ok, changes seem simple enough. But, with this PR we have a new concept (I think): changing the visitor without changing grammar. When we make grammar + visitor updates, we run regression testing on all checks with all projects.  What do you think about this?

Reply:
I think it is pointless to run check regression in this case.

The most important thing with this type of change is to first ask did the tree change? My ANTLR regression proves that the tree did not change for atleast the files it scans in our repo. I consider our most complex repo in our projects to test on and when I am regression hunting, majority times now I find cases in our own repo.

If the tree has absolutely no changes, how will the checks be impacted to show a change in regression? I believe this is the first time we have done what I believe to be a no change to regression.

If you are thinking this is not enough, then maybe your real concern is there is not enough ANTLR regression since I ran just the 1 project? I can run it on another of your choosing, but please note it took 77 minutes to run ANTLR regression on this 1. Bigger repos like openjdk will take hours and lock me out until it completes, which I am unsure how long that will be as I avoid bigger repos. I can also run check regression if you want, but I think it won't show anything (besides exceptions).

Also to note, we are now 100% with mutation on AST which should mean there is nothing outside our repo for our ANTLR besides these small little data points of concern.

Reply:
Side note for the future, maybe we should consider an alternative like ANTLR hashcode regression and see if that can run faster than full ANTLR print regression since we are not printing out each node, one by one. This would have to be a special custom Check, which we have not introduced into regression yet.

Another factor in the slowdown is we do javadoc ANTLR as well in the same run, so maybe that should be separated between Java only and Java and Javadoc. We would only be able to do Javadoc only printing regression with another special custom Check.

Reply:
we can run with and without to be safe. as it is not javadoc it will not be very slow, so activation of all Checks in default (or any config, for example our config) will be ok.

Reply:
I am attempting to run regression.

Reply:
Regression: http://rveach.no-ip.org/checkstyle/regression/332/

No differences.

Reply:
Thanks for check regression test.